src/core_modules/capture-core/components/DataEntry/dataEntryField/internal/dataEntryField.utils.js

```diff
@@ -24,10 +24,10 @@ export function getValidationError(value: any, validatorContainers: ?Array<Valid
             return false;
         }
 
-        message = (result && result.errorMessage) || (result && result.message) || validatorContainer.message;
+        errorMessage = (result && result.errorMessage) || validatorContainer.errorMessage;
```

Comment:
Should the change to `errorMessage` also be applied to [validateValue](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/utils/validation/validateValue.js#L50) used by the FormBuilder.component.js? Something like:
```
return {
   message: (result && result.errorMessage) || currentValidator.errorMessage,
   ....
};
```
WDYT?

Reply:
I didn't change the validators in [getValidators](https://github.com/dhis2/capture-app/blob/5da612691342624007789d0d6befca608cca91ba/src/core_modules/capture-core/utils/validation/getValidators.js#L242) file. They are still using `message`.